### DISCUSSION TITLE
Best installation method for a command line use

### DISCUSSION THREAD

**Discussion by luke-on-security** [cedricbonhomme/Stegano#42] [Q&A] [answered]
I would like to know how to install the app system wide ?

thanks !

**Comment by cedricbonhomme** [comment, marked as answer]
Hey again !


the app is designed for this use as well and is available on pypi !

So you can simple use pip or pipx. I rreally recommend pipx. The app will be available for the user (user wide), and installed in a dedicated Python vitualenv. So no dependendies issue.

**Reply by luke-on-security** [reply]
thanks for the quick answer ! I will look at pipx !
